Terran-One/cw-simulate-ui
# (WIP) Load sample contract with schema

## Issue link

[WL-XXX](https://terran-one.atlassian.net/browse/WL-XXX)

## Description

Load sample contract with schema

## Test steps

1. Load sample contract 
2. Verify that schema is pulled from R2
3.
